doc/releases/changelog-dev.md

```diff
@@ -50,6 +50,11 @@
 
 <h3>Deprecations</h3>
 
+* `with qml.tape.QuantumTape()` is deprecated. Please initialize `QuantumTape` instances directly instead.
```

Comment:
Given the size of this deprecation, we should use a code example as well.

Reply:
in the changelog _and_ the deprecation file (where I added one already)?